x-pack/plugins/secops/common/graphql/types.ts

```diff
@@ -108,6 +138,16 @@ export namespace SourceResolvers {
     Parent = any,
     Context = any
   > = Resolver<R, Parent, Context>;
+  export type GetSuricataEventsResolver<
+    R = (SuricataEvents | null)[] | null,
```

Comment:
Is it possible to say this always resolves to an empty collection, rather than a collection that may be `null`?

Reply:
I like to make the difference between [] there is no data and null it does not exist. and this can happen if you do not set up your config the right way

Reply:
I suppose it's a matter of style, but functions that can return `null` or `undefined` in addition to returning an empty collection or a non-empty collection require consumers of said function to add an explicit check before using the return value, and when that get's missed, the result is bugs.

My preference is to make use of the type system to make code safer while removing the need for explicit checks. Not a mandatory change, but perhaps we should discuss it as a team.